<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graf_form</class>
 <widget class="QDialog" name="graf_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QRadioButton" name="hGrafBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <stylestrategy>PreferAntialias</stylestrategy>
         <fontweight>Light</fontweight>
        </font>
       </property>
       <property name="text">
        <string>Est치mos trabajando en esta funci칩n</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="ID_bsc_btn">
       <property name="text">
        <string>ID de empleado</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Encontrar por:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="bGrafBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <stylestrategy>PreferAntialias</stylestrategy>
         <fontweight>Light</fontweight>
        </font>
       </property>
       <property name="text">
        <string>Est치mos trabajando en esta funci칩n</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="Btn_Ok">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Datos del empleado a encontrar</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="line_id">
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string notr="true" comment="Ingrese el ID, por ejemplo RM000" extracomment="000"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Buscar Empleados</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
